ansible/druid-ingestion.yml

```diff
@@ -1,6 +1,56 @@
 ---
+- hosts: rollup-overlord
+  become: yes
+  pre_tasks:
+   - name: set_fact
+     set_fact:
+       single_cluster: true
+     tags: populate_var
+
+   - name: check for hosts
+     set_fact:
+       single_cluster: false
+     when: "'rollup-overlord' in groups"
+     tags: populate_var
+
+   - name: debug
+     debug:
+       msg: "{{single_cluster}}"
+     tags: populate_var
+  roles:
+    - role: druid-ingestion
+      vars:
+        ingestion_task_names: "{{ rollup_ingestion_task_names }}"
+        overlord_host: "{{ rollup_overlord_host }}"
+      when: not single_cluster
 
-- hosts: "{{ cluster }}-overlord"
+- hosts: raw-overlord
   become: yes
+  pre_tasks:
```

Comment:
This wont be required after implementing previous comment

Reply:
Ansible is not taking the variable from previous play.